resources/views/dashboard.blade.php

```diff
@@ -109,7 +109,7 @@
         methods: {
             connect() {
                 this.pusher = new Pusher(this.app.key, {
-                    wsHost: window.location.hostname,
+                    wsHost: this.app.host.length === 0 ? window.location.hostname : this.app.host,
```

Comment:
```suggestion
                    wsHost: this.app.host === null ? window.location.hostname : this.app.host,
```

Checking on the length of null results in an error.

Reply:
Sorry, my fault, my first draft was to add an empty string as default!